src/dlr.cc

```diff
@@ -104,6 +104,21 @@ extern "C" int SetDLRInputTensor(DLRModelHandle* handle, const char* name, void*
   API_END();
 }
 
+extern "C" int SetDLRInputTensorZeroCopy(DLRModelHandle* handle, const char* name, void* tensor) {
+  API_BEGIN();
+  DLRModel* dlr_model = static_cast<DLRModel*>(*handle);
+  DLRBackend backend = dlr_model->GetBackend();
+  CHECK(backend == DLRBackend::kTVM)
+      << "model is not a TVMModel. Found '" << kBackendToStr[static_cast<int>(backend)]
+      << "' but expected 'tvm'";
```

Comment:
Do you wanna support RelayVM backend in this PR or a separate one? It looks like we can call the `FromDLPack` API on TVM NDarrays to do a "zero copy" on DLManagedTensors too:
https://github.com/apache/tvm/blob/81d9f11ab87dc3ef5fc906aa6ca23737885f7b27/src/runtime/ndarray.cc#L199
on the `inputs_[index]` in RelayVM backend:
https://github.com/neo-ai/neo-ai-dlr/blob/master/src/dlr_relayvm.cc#L281